### ISSUE TITLE
autocomplete for object in assoc array 

### ISSUE THREAD

**Issue by ooghry** [klesun/deep-assoc-completion#71]
Hi there
In the `get` method, $period['period']->end is DateTime but there is no suggestion for its methods.

```php
<?php

class test
{
    public function my()
    {
        return [
            'period' => new \DatePeriod(
                new \DateTimeImmutable('now'),
                new \DateInterval('P0Y'),
                new \DateTimeImmutable('P3DAY')
            )
        ];
    }

    public function get()
    {
        $period = $this->my();
        //$period['period']->end is DateTime
        //but there is no suggestion for its methods.
        echo get_class($period['period']->end);
        return $period['period']->end->getTimeStamp();
    }
}

$test = new test;
echo $test->get();
```

**Comment by klesun** [comment]
Look like you are right, thanks, I'll try to fix this today.

**Comment by klesun** [comment]
This is possibly because there are 2 definitions of `\DatePeriod` class bundled in IDEA:
```
IDEA_DIR/plugins/php/lib/php.jar!/stubs/date/date_c.php
IDEA_DIR/config/plugins/php/lib/php.jar!/stubs/date/date_c.php
```

**Comment by ooghry** [comment]
Is there any hope you can fix?

**Comment by klesun** [comment]
yeah, sorry, was too busy yesterday, but it's an easy task

**Comment by klesun** [comment]
Sorry for the waiting, here is the fix - (2019.04.15.001)[https://plugins.jetbrains.com/plugin/9927-deep-assoc-completion/update/60925]
![image](https://user-images.githubusercontent.com/5202330/56134498-374fb780-5f7e-11e9-82bd-4660b3578d10.png)

Apparently plugin totally ignored field type doc comment...
